lib/commands/team.js

```diff
@@ -0,0 +1,229 @@
+/* eslint-disable standard/no-callback-literal */
+
+const figgyPudding = require('figgy-pudding')
+const libteam = require('libnpmteam')
```

Comment:
Please move the `libnpmteam` require inside the individual commands to help reduce overall startup times. The other three can stay because pretty much every single other command is going to use them, so they won't make any difference.

Reply:
`const libteam = require('libnpmteam')` moved into every single command.